src/multiplayer-servers/multiplayer-services/server-protection/protocols.md

```diff
@@ -0,0 +1,68 @@
+# Ports and Protocols
+
+Game server ports are often vulnerable to attacks. 
+GameFabric offers Protocols to protect your game server traffic with powerful SteelShield mitigations.
+
+## Pre-requisites
+
+In order to protect the your game server traffic, your GameFabric installation requires:
+
+- Protection support enabled, usually indicated by the presence of the Protection/SteelShield navigation item.
+- Sites that support SteelShield, e.g. bare-metal.
+
+## Mitigations
+
+SteelShield provides mitigations that are tailored to check and let through only specific game or network traffic.
+To make use of the pre-defined mitigations you must create a new Protocol.
+
+## Protocols
+
+A Protocol – sometimes referred to as Protection Protocol, to distinguish it from a 
+network protocol such as UDP or TCP –
+describes the application protocol used by your game traffic or service.
+
+When you create a new Protocol, you must select a Mitigation and a network protocol.
+The network protocol that you select must be compatible with the Mitigation and their network protocol in order to be selectable.
+
+![create-protocol.png](images/create-protocol.png)
+
+::: warning
+You can restrict your Protocol to only support one UDP or TCP, even though the Mitigation would allow both.
+The other way around – allow both where only one is supported by the Mitigation – is not allowed.
+:::
+
+Once the Protocol is created, it is available to all ArmadaSets, Armadas, Formations and Vessels.
+Nonetheless you need to actively select a Protection Protocol for each game server port of these entities.
+
+## Protect game server ports
+
+Any ArmadaSet, Armada, Formation or Vessel requires at least one game server port. 
+This port is usually to allow communication between your game server and your connecting game clients.
+It is however common to use multiple ports, one for the game engine, one for query services and so on.
+For each of them you can apply different Protocols.
+
+All ports are potentially vulnerable to attack, so by setting a Protection Protocol,
+SteelShield applies a mitigation action to the traffic on these ports.
+
+When the port is unprotected (`none`), SteelShield forwards any traffic without checking or filtering it.
+
+You can find the Protocol assignment to the right of the port specification under 
+Vessel » Settings » Containers » Container Ports. It is the same for ArmadaSets, Armadas and Formations.
```

Comment:
This isn't specific to Vessels, and I would suggest not including a path here, since it is likely that section names and structure will change and we will not think to update the docs (for reference, see Keycloak documentation, all mentions of where to do things are wrong and they make me want to throw my computer out of the window)

Reply:
Yeah I am in a struggle between showing the reader where he can actually configure it, and not being required to update the docs all the time to reflect the current GF nav structure. 

Anyway cropped off the sentence a bit. WDYT?